eZ/Publish/Core/FieldType/Time/Type.php

```diff
@@ -154,6 +154,25 @@ public function fromHash($hash)
     }
 
     /**
+     * Returns if the given $value is considered empty by the field type.
+     *
+     *
+     * @param \eZ\Publish\Core\FieldType\Value $value
+     *
+     * @return bool
+     */
+    public function isEmptyValue(SPIValue $value)
+    {
+        if (empty($value)) {
```

Comment:
Is there some inconsistency here inherited from the other field types? Type hints requires instance of SPIValue here, so null or string is not possible right?